src/app/client/src/app/app.component.ts

```diff
@@ -493,6 +493,9 @@ export class AppComponent implements OnInit, OnDestroy {
    */
   public setDeviceId(): Observable<string> {
       return new Observable(observer => this.telemetryService.getDeviceId((deviceId, components, version) => {
+        if (this.utilService.isDesktopApp) {
```

Comment:
There should be else condition otherwise it  overrides with fingerPrint js device id